manifests/a/asdfjkl/YAHB/1.0.8.0/asdfjkl.YAHB.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: asdfjkl.YAHB
+PackageVersion: 1.0.8.0
+PackageLocale: en-US
+Publisher: asdfjkl
+PackageName: YAHB
+License: GNU GPL 3
+ShortDescription: YAHB is a deduplicating file copy tool, intended for backup use. Deduplication works on the file-level with NTFS hardlinks.
+ManifestType: defaultLocale

```

```suggestion
Publisher: asdfjkl
PublisherUrl: https://github.com/asdfjkl
PublisherSupportUrl: https://github.com/asdfjkl/YAHB/issues
Author: Dominik Klein
PackageName: YAHB
PackageUrl: https://github.com/asdfjkl/YAHB
License: GPL-3.0
LicenseUrl: https://github.com/asdfjkl/yahb#GPL-3.0-1-ov-file
ShortDescription: Deduplicating File-Copy/Backup Tool
Description: YAHB is a deduplicating file copy tool, intended for backup use. Deduplication works on the file-level with NTFS hardlinks.
Moniker: yahb
ReleaseNotesUrl: https://github.com/asdfjkl/yahb/releases/tag/v1.0.8
ManifestType: defaultLocale
```
It's generally better to include as much metadata as possible